server/routes/selectAssociatedStaff/selectAssociatedStaff.ts

```diff
@@ -1,27 +1,29 @@
 import url from 'url'
 import { Request, Response } from 'express'
 import { FormError } from '../../@types/template'
-import UserService, { StaffSearchByName } from '../../services/userService'
+import UserService from '../../services/userService'
 import validateForm from './staffSearchValidation'
+import PlaceOnReportService, { StaffSearchByNameEnhanced } from '../../services/placeOnReportService'
 
 type PageData = {
   error?: FormError
-  searchResults?: StaffSearchByName[]
-  searchFirstName: string
-  searchLastName: string
+  searchResults?: StaffSearchByNameEnhanced[]
```

Comment:
I would prefer this to have a name that represents what it contains - so `StaffSearchByWithDetails` or something?